layer_tests/tracing/src/test_api_ltracing.cpp

```diff
@@ -943,6 +943,42 @@ TEST_F(
   ASSERT_EQ(initial_result, zeCommandListReset(command_list));
 }
 
+TEST_F(
+    LTRACING_TEST_NAME,
+    GivenEnabledTracerWithCalltoZetAPIThenSuccessIsReturned) {
```

Comment:
Could you edit the test names to have a 'When' portion?

Reply:
done, updated